composer.json

```diff
@@ -30,7 +30,7 @@
         "ext-openssl": "To use Security::encrypt() or have secure CSRF token generation."
     },
     "require-dev": {
-        "phpunit/phpunit": "*",
+        "phpunit/phpunit": "~5.7.0",
```

Comment:
Yes, that is exactly the problem. The latest stable version for php 7 requires code changes that are incompatible with previous versions.